src/main/java/de/dennisguse/opentracks/content/provider/ContentProviderUtils.java

```diff
@@ -205,15 +210,10 @@ public void deleteTrack(Context context, @NonNull Track.Id trackId) {
         return tracks;
     }
 
-    public List<Track> getTracks(List<Track.Id> trackIds) {
-        if (trackIds == null || trackIds.isEmpty()) {
-            return getTracks();
-        }
-
-        String selection = String.format(TracksColumns._ID + " IN (%s)", TextUtils.join(",", Collections.nCopies(trackIds.size(), "?")));
-        String[] selectionArgs = trackIds.stream().map(id -> Long.toString(id.getId())).toArray(String[]::new);
+    public List<Track> getTracks(ContentProviderSelectionInterface selection) {
```

Comment:
Is this @NonNull?

Reply:
We already have `getTracks()` method so if someone uses `getTracks(selection)` s-he should pass a selection.